src/app/components/GoogleAuth.tsx

```diff
@@ -27,7 +27,7 @@ const styles = {
     bg-white
     rounded-full
     border
-    border-[rgba(15,20,25,0.1)] dark:border-[#333639]
+    border-primaryBorder-light dark:border-primaryBorder-dark
```

Comment:
I notice there is almost everytime (if not every) light theme colors used alongside with their counterpart in dark theme. What are you thinking to make it more laconic? It's not really urgent or necessary but good thing to think about.